### PR TITLE
to Reggie for Testing

### PR DESCRIPTION
**Pull request by Copenbacon**

### CODE REVIEW

File: src/setup.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from setuptools import setup
+
+setup(
+    name="Mailroom",
+    description="An implementation of the Mailroom Module",
+    version=0.1,
+    author=["Conor Clary", "Regenal Grant"],
+    author_email=["sclary50@gmail.com", "regenal@mac.com"],
+    license="MIT",
+    package_dir={'': './'},
```
Comment (WillWeatherford) [inline src/setup.py:13]: This right here is a bit of a problem. The value in that key/value pair is the directory of code which the installer will try to install. This should be your `src` directory, not its parent.

File: src/setup.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    py_modules=['main'],
+    install_requires=['numpy', 'faker', 'inquirer'],
```
Comment (WillWeatherford) [inline src/setup.py:15]: Are you actually using these packages? If not, remove them, because it takes time and space to install them.

Comment (Copenbacon) [reply src/setup.py:15]: done

File: src/setup.py (hunk 1 above)
Comment (Copenbacon) [reply src/setup.py:13]: done

**Pull request merged**